packages/block-editor/src/components/inserter/media-tab/media-preview.js

```diff
@@ -203,7 +205,12 @@ export function MediaPreview( { media, onClick, composite, category } ) {
 	const onMouseLeave = useCallback( () => setIsHovered( false ), [] );
 	return (
 		<>
-			<InserterDraggableBlocks isEnabled={ true } blocks={ [ block ] }>
+			<InserterDraggableBlocks
+				isEnabled={ true }
+				blocks={ [ block ] }
+				label={ blockType?.title }
+				icon={ blockType?.icon }
```

Comment:
If an icon is provided the `label` is not used anywhere, so we wouldn't need it.

Reply:
Sorry for the ignorance, but will a block always have an icon? I was thinking about extensibility and the case where some third party wants to use this, and the block doesn't have an icon.